kobo/apps/trash_bin/tasks/attachment.py

```diff
@@ -79,6 +89,62 @@ def schedule_auto_attachment_cleanup_for_users():
         auto_delete_excess_attachments(counter.user.pk)
 
 
+@celery_app.task(
+    autoretry_for=(
+        TimeLimitExceeded,
+        TrashTaskInProgressError,
+        SoftTimeLimitExceeded,
+    ),
+    retry_backoff=60,
+    retry_backoff_max=600,
+    max_retries=5,
+    retry_jitter=False,
+    queue='kpi_low_priority_queue',
+    soft_time_limit=settings.CELERY_LONG_RUNNING_TASK_SOFT_TIME_LIMIT,
+    time_limit=settings.CELERY_LONG_RUNNING_TASK_TIME_LIMIT,
```

Comment:
Do you think it's necessary. Moving files to trash should be fast enough to run under 30 minutes (the default)?